src/screens/TopicPageScreen/elements/BottomSheetFollow.tsx

```diff
@@ -89,6 +82,9 @@ const BottomSheetFollow = forwardRef((props: BottomSheetFollowProps, ref: Ref<RB
     setIsFollow(false);
     setMemberCount(memberCount - 1);
     onClose();
+    console.log('topicName', topicName);
+    updateFollowTopicDiscoveryContext(false, {topic_id: topicName}, true);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a console log statement on line 85 which might have been used for debugging purposes. It's generally a good practice to remove such statements before committing the code to avoid cluttering the console output in a production environment.

```diff
- console.log('topicName', topicName);
```

Also, similar to the previous comment, consider improving the readability of the `updateFollowTopicDiscoveryContext` function call.

```diff
- updateFollowTopicDiscoveryContext(false, {topic_id: topicName}, true);
+ updateFollowTopicDiscoveryContext({isFollowing: false, topic: {id: topicName}, shouldUpdate: true});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->